pkg/postgres/configuration.go

```diff
@@ -428,6 +428,7 @@ var (
 			{130000, MajorVersionRangeUnlimited}: {
 				"wal_keep_size":      "512MB",
 				"shared_memory_type": "mmap",
+				"wal_level":          "logical",
```

Comment:
I think we need to set `wal_level` to `logical` by default in every PostgreSQL version. Logical replication is always supported for PostgreSQL versions that we use in CNPG.